### PR TITLE
Add logout link

### LINKED ISSUE #54: Add ability to log out from the application
## Is your feature request related to a problem? Please describe.
Currently there is no way to log out of the application once logged in. 

## Describe the solution you'd like
Add the ability to logout. For now, this could be a link in the header.

### PR DESCRIPTION
**Pull request by p-galligan**
Add link to log out of application. Automatically redirects to login page. Fixes #54 

I think there's an accessibility/styling conversation to be had here. The contrast between the links in the navbar and the color of the header isn't great. We may want to introduce some custom styling.

### CODE REVIEW

**Comment by HaSistrunk** [comment]
This is a little funky on mobile. I'm wondering if we might simplify into just one link that would indicate both that someone is logged in, and allow them to logout? Like:
```
      {% if request.user.is_authenticated %}
        <div class="navbar-nav my-lg-0">
          <span class="nav-item">
            <a class="nav-link" href="{% url 'logout' %}">Log out {{request.user.username}}</a>
          </span>
        </div>
      {% endif %}
```

Good call about the colors in the nav. I'm not a fan of that dark gray from Bootstrap either. I can look at some alternatives as part of issue #35 .

**Comment by p-galligan** [comment]
Good idea. I've integrated the changes.

File: assign_rights/templates/navbar.html [hunk 1]
```diff
@@ -20,12 +20,11 @@
       </ul>
       <div class="form-inline my-2 my-lg-0">
```
Comment (HaSistrunk) [inline assign_rights/templates/navbar.html:21]:
```suggestion
```

**Review by HaSistrunk — APPROVED** [review]

**Review by helrond — APPROVED** [review]
Looks good to me!

In terms of contrast, the Bootstrap [color utilities](https://getbootstrap.com/docs/4.5/utilities/colors/) may be useful.

**Review by bonniegee — APPROVED** [review]
Looks good!

**Pull request merged**